bot/cogs/cogs.py

```diff
@@ -0,0 +1,196 @@
+# coding=utf-8
+import os
+
+from discord import Embed, ClientException, Colour
+from discord.ext.commands import AutoShardedBot, Context, command
+
+from bot.constants import (
+    MODERATOR_ROLE, ADMIN_ROLE, DEVOPS_ROLE, OWNER_ROLE, GREEN_CHEVRON, RED_CHEVRON
+)
+
+from bot.decorators import with_role
+from bot.utils import paginate
+
+
+class Cogs:
+    """
+    Cog management commands
+    """
+
+    def __init__(self, bot: AutoShardedBot):
+        self.bot = bot
+        self.cogs = {}
+
+        # Load up the cog names
+        for filename in os.listdir("bot/cogs"):
+            if filename.endswith(".py") and "_" not in filename:
+                if os.path.isfile(f"bot/cogs/{filename}"):
+                    cog = filename[:-3]
+
+                    self.cogs[cog] = f"bot.cogs.{cog}"
+
+        # Allow reverse lookups by reversing the pairs
+        self.cogs.update({v: k for k, v in self.cogs.items()})
+
+    @command(name="cogs.load()", aliases=["cogs.load", "load_cog"])
+    @with_role(MODERATOR_ROLE, ADMIN_ROLE, OWNER_ROLE, DEVOPS_ROLE)
+    async def load_command(self, ctx: Context, cog: str):
+        cog = cog.lower()
+
+        embed = Embed()
+        embed.colour = Colour.red()
+
+        embed.set_author(
+            name="Python Bot (Cogs)",
+            url="https://github.com/discord-python/bot",
+            icon_url="https://raw.githubusercontent.com/discord-python/branding/master/logos/logo_circle.png"
+        )
+
+        if cog in self.cogs:
+            full_cog = self.cogs[cog]
+        elif "." in cog:
+            full_cog = cog
+        else:
+            full_cog = None
+            embed.description = f"Unknown cog: {cog}"
+
+        if full_cog == "bot.cogs.cogs":
+            embed.description = "You may not reload the cog management cog!"
```

Comment:
this will never happen. The cog can't be unloaded, and if it was, you couldn't do this anyway cause it'd be unloaded.

Reply:
Got this one already

Reply:
wait, this one actually could happen. it's the load_cog one that couldn't.  anyway it seems fixed now.